### PR TITLE
chore: Added WorkflowTemplate test

### PR DESCRIPTION
**Pull request by simster7**
Closes: https://github.com/argoproj/argo/issues/1953

### CODE REVIEW

**Comment by simster7** [comment]
@alexec Added CLI and API tests

File: test/e2e/cli_test.go [hunk 1]
```diff
@@ -69,6 +69,36 @@ func (s *CLISuite) TestRoot() {
 	})
 }
 
+func (s *CLISuite) TestTemplate() {
+
```
Comment (alexec) [inline test/e2e/cli_test.go]: `template lint`?

File: test/e2e/argo_server_test.go [hunk 2]
```diff
@@ -738,13 +738,52 @@ func (s *ArgoServerSuite) TestWorkflowTemplates() {
 			Equal(1)
 	})
```
Comment (alexec) [inline test/e2e/argo_server_test.go]: add a lint test?

**Comment by simster7** [comment]
@alexec Added lint tests

**Comment by simster7** [comment]
Closed in favor of #1989

**Pull request closed without merging**